docs/root/configuration/runtime.rst

```diff
@@ -79,6 +93,35 @@ old tree to the new runtime tree, using the equivalent of the following command:
 
 It's beyond the scope of this document how the file system data is deployed, garbage collected, etc.
 
+.. _config_runtime_layering:
+
+Layering
+--------
+
+The runtime can be viewed as virtual filesystem consisting of multiple layers:
+
+1. :ref:`Static bootstrap configuration <envoy_api_field_config.bootstrap.v2.Runtime.base>`
+2. :ref:`Disk file system <config_runtime_file_system>`
+3. Disk file system override
+4. :ref:`Admin interface overrides <operations_admin_interface_runtime_modify>`
+
+with higher layers overriding corresponding values in lower layers.
+
+.. _config_runtime_proto_json:
+
+Protobuf and JSON representation
```

Comment:
Example yaml in docs or our sample configs?

Reply:
+1